cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/StringExpressionFactoryType.java

```diff
@@ -14,24 +14,14 @@
  * the License.
  */
 
-package io.cdap.cdap.etl.api.aggregation;
+package io.cdap.cdap.etl.api.relational;
```

Comment:
This renames looks weird. Could you please check why git picks this set of changes as renames? It's hard to follow

Reply:
Not sure why git picks it as a rename. I've deleted the `GroupByFields` class since we're not using Field classes any more and added the `StringExpressionFactoryType` class (technically it's a rename of `StringExpressionFactory` since it's actually an implementation of `ExpressionFactoryType`), but oddly the diff treats it as a file rename rather than two separate file delete and file addition operations.

If anything the diff should have treated `StringExpressionFactoryType` as a rename of `StringExpressionFactory` and then flagged the new `StringExpressionFactory` class as a new file, as that was the actual intent of the PR.